frontend/packages/dev-console/src/components/BuildconfigPage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { match as RMatch } from 'react-router';
+import { BuildsPage } from '@console/internal/components/build';
+import DefaultPage from './DefaultPage';
+
+export interface BuildconfigPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+const BuildconfigPage: React.FC<BuildconfigPageProps> = (props) => {
+  const namespace = props.match.params.ns;
+  return (
+    <React.Fragment>
+      <Helmet>
+        <title>Builds</title>
+      </Helmet>
+      {namespace ? (
+        <BuildsPage {...props} />
```

Comment:
Title needs to be `Builds` but the `BuildConfigsPage` has a fixed title of `Build Configs`.